### DISCUSSION TITLE
About  support for new Unity versions

### DISCUSSION THREAD

**Discussion by zhangjiequan** [aelurum/AssetStudioMod#12] [General]
I believe I've potentially extended support to Unity 2022.2 and 2022.3. 
You might want to give it a try: https://github.com/zhangjiequan/AssetStudio.

**Comment by aelurum** [comment]
Well, AssetStudioMod also has support for Unity 2022.2 and 2022.3
I just want to do more tests before changing a readme file

**Reply by IDBT** [reply]
By the way, I would like to ask the developers, are there any plans to support Unity6?

**Reply by aelurum** [reply]
The latest builds of AssetStudioMod already support Unity6.

**Reply by IDBT** [reply]
Dear aelurum,
Thanks for the quick reply.

I immediately loaded the asset bundle for the game I was interested in and was unable to load some assets.
The warning message was as follows.

> [Warning] Unable to load object
>[Warning] Assets ******
>[Warning] Path ***\StreamingAssets\aa\StandaloneWindows64\e9ba6b76be15d47c76a2cc25b9ae8cf1.bundle
>[Warning] Type SkinnedMeshRenderer
>[Warning] PathID ******
>[Warning] System.IO.EndOfStreamException: Unable to read beyond the end of the stream.
>[Warning]    at AssetStudio.EndianBinaryReader.ReadInt64() in L:\GIT\AssetStudioMod\AssetStudio\EndianBinaryReader.cs:line 52
>[Warning]    at AssetStudio.SkinnedMeshRenderer..ctor(ObjectReader reader) in >L:\GIT\AssetStudioMod\AssetStudio\Classes\SkinnedMeshRenderer.cs:line 14
>[Warning]    at AssetStudio.AssetsManager.ReadAssets() in L:\GIT\AssetStudioMod\AssetStudio\AssetsManager.cs:line 598
>[Warning]

"*****" is the part that have dealed with privacy.

The version of the game I loaded was 6000.0.32f1, so I think that's as far as the latest version supports.
If there is any remedy I can take, please let me know.

**Reply by aelurum** [reply]
This is a release version.
You can download the latest build by clicking the "Download latest build" button on the home page.
![image](https://github.com/user-attachments/assets/63201a51-fe71-43b6-8f54-594f4addd6a2)

Also, there is always a chance that the asset structure has been modified by game devs.

**Reply by IDBT** [reply]
Thanks for your answer.
I downloaded the latest version again and loaded the same AssetBundle and it worked fine without errors or warnings.
Thank you so much for your patience in answering my question.
